packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/feature_splash/blocs/splash_bloc.dart

```diff
@@ -64,7 +64,13 @@ class SplashBloc extends $SplashBloc {
       _navigationBloc.events.goToLocation(_redirectLocation!);
     } else {
       await _authService.isAuthenticated()
-          ? _navigationBloc.events.go(const CounterRoute())
+          ? 
+          {{#enable_feature_counter}}
+          _navigationBloc.events.go(const CounterRoute())
```

Comment:
Considering the counter will become an optional feature I believe we can add a new one called dashboard, which will be in the first position in the nav menu as well as the default page, which the logged users will see.